CardinalKit-Example/CardinalKit/Library/Networking/CKSendHelper.swift

```diff
@@ -154,6 +157,21 @@ class CKSendHelper {
         }
     }
     
+    static func createNecessaryDocuments(path: String){
```

Comment:
Hi @EstebanRamosFractal - 

Can we add a comment header describing this function? 

Is it possible to call `createNecessaryDocuments` once from the `init()` function (override) of this `CKSendHelper` class?

Reply:
This function creates the necessary documents in firebase by adding a data to avoid virtual documents, virtual documents in firebase make requests and filters impossible, so it is necessary to call this function every time I need to send data to firebase